docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -14,17 +14,17 @@ include::_attributes.adoc[]
 :extensions: io.quarkus:quarkus-tls-registry
 
 The TLS registry is a Quarkus extension centralizing the TLS configuration for the application.
-It allows to define the TLS configuration in a single place and to reference it from multiple places in the application.
+It allows you to define the TLS configuration in one place and reference it from multiple places in the application.
```

Comment:
In _IBM Style_, the "Anthropomorphism" guideline suggests refocusing on the user:

> Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions. See specific guidance for Marketing.
> 
> - [Avoid] Product focus: The Delivery Information window allows you to specify the name of the sender.
> - [Prefer] User focus: In the Delivery Information window, specify the name of the sender. "

```suggestion
You can use the TLS registry to define the configuration in one place and reference it from multiple places in the application.
```

Reply:
In _IBM Style_, also see:
> allow verb
> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct,
> user-focused alternative such as “you can use the product to…” or “with this product, users can…”.